src/lib/fs/path.h

```diff
@@ -20,11 +21,21 @@
 #define PATH_SEPARATOR "/"
 #endif
 
+#ifdef _WIN32
+#define IS_GLOB_CHAR(s,i) (((s)[(i)]) == '*' || ((s)[(i)]) == '?')
```

Comment:
You don't allow escaped glob chars on Windows?
Where is this difference documented?

Reply:
Are there tests for escaping on Windows?
What happens?

Reply:
Escaping glob chars is not supported by Windows because Windows does not support * or ? in filenames. This will be documented.

The escape character is a blackslash which is a path separator on Windows. What happens when someone migrates torrc to Windows depends where the glob is:
- If there is only one escaped character and it is in the last path component, the escape character will be interpreted as a path separator and the escaped glob will not be escaped. I do not think is possible to determine if this is intentional by the user or if it is a mistake. However, I believe this is a very unlikely situation as it is unlikely that someone is using ? or * on their config file names.
- In any other case, there should be an error.

Reply:
Documentation added in bf4c5f7.